packages/memory/integration-tests/src/working-memory.test.ts

```diff
@@ -56,7 +56,7 @@ describe('Working Memory Tests', () => {
   describe('Working Memory Test with Template', () => {
     beforeEach(async () => {
       // Create a new unique database file in the temp directory for each test
-      const dbPath = join(await mkdtemp(join(tmpdir(), `memory-working-test-`)), 'test.db');
+      const dbPath = join(await mkdtemp(join(tmpdir(), `memory-working-test-${Date.now()}`)), 'test.db');
```

Comment:
mkdtemp does a dynamic path, so you dont need Date.now
```suggestion
      const dbPath = join(await mkdtemp(join(tmpdir(), `memory-working-test-`)), 'test.db');
```

Reply:
It was the same path for all the tests causing weird issues

Reply:
yeah we were seeing that without Date.now() it was re-using the same db across tests. We should revisit it and figure out why (or use MockStore) separately from this PR since we want to get this into the release today